app/controllers/application_controller.rb

```diff
@@ -3,6 +3,17 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  rescue_from CanCan::AccessDenied do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.